tests/test_mailcow.py

```diff
@@ -10,10 +10,11 @@ def test_get_users(self, mailcow):
 
     def test_add_del_user(self, mailcow):
         addr = "pytest.%s@x.testrun.org" % (randint(0, 999),)
-        mailcow.add_user_mailcow(addr, "asdf1234")
+        mailcow.add_user_mailcow(addr, "asdf1234", "pytest")
 
         user = mailcow.get_user(addr)
         assert user.addr == addr
+        assert "mailadm:pytest" in user.tags
```

Comment:
```suggestion
        assert "pytest" == user.token
```
This way "add_user_mailcow" allows to add a single token, and the user object exposes the "token". 
Giving access to or managing "tags" is then a separate operation (that this PR does not need to introduce neccessarily).

Reply:
in other words, each mailcow user can then have None or exactly one token associated.